src/common/integration_test_base_class.py

```diff
@@ -0,0 +1,79 @@
+# standard library
+import unittest
+
+# third party
+import mysql.connector
+
+# first party
+from delphi.epidata.client.delphi_epidata import Epidata
+from delphi.epidata.server._limiter import limiter
+
+
+class BasicIntegrationTest(unittest.TestCase):
```

Comment:
```suggestion
class IntegrationTestBase(unittest.TestCase):
```
^ name suggestion to hopefully make it obvious that this is a "base" class.  (im also a little bit tempted to call it `DelphiTestBase`, but thats up to you)

---

this class has a number of things in common with our other `unittest.TestCase` subclass / test base class helper: [`CovidcastBase`](https://github.com/cmu-delphi/delphi-epidata/blob/f7da6598a810be8df5374e3a71512c631c3a14f1/src/acquisition/covidcast/test_utils.py#L156).  `CovidcastBase` would best be redesigned as a subclass of the common `IntegrationTestBase` that then adds `epimetric_*` table truncations in setUp() and the additional row helper methods.

would it be difficult to combine them?

NOTE: maybe the suggested changes of combining / refactoring with `CovidcastBase` could be its own PR (if at all) after this one is merged.  also, `CovidcastBase` has a number of things i think should change, such as changing the name to `CovidcastTestBase` and relocating the file to `src/common/` instead of being under `src/acquisition/`.

Reply:
Yeah, that renaming is reasonable.

Yeah, I have also thought about combining this class with CovidcastBase, but it will be better to do that in a separate PR(after this one is merged).